### DISCUSSION TITLE
请问 README 插图中的引用是本仓库的功能支持的效果吗

### DISCUSSION THREAD

**Discussion by function2-llx** [Keldos-Li/typora-latex-theme#48] [Q&A] [answered]
如图，如果是的话很想使用这个功能。

![image](https://user-images.githubusercontent.com/38486514/143458519-211c1256-e0d4-43ec-b36e-4b83655ed828.png)

**Comment by Keldos-Li** [comment, marked as answer]
README 插图中的文件存放在 `Supplemental` 文件夹：https://github.com/Keldos-Li/typora-latex-theme/blob/main/Supplemental/essay-template.md

其实是手动的“引用”，就是上标了 [1] ……

**Reply by function2-llx** [reply]
好的，谢谢！
